src/components/VmActions/index.js

```diff
@@ -107,11 +107,17 @@ const VmActions = ({ vm, isOnCard = false, onGetConsole, onShutdown, onRestart,
     cancelText: 'Cancel',
   }
 
+  let consoleStatus = 'Get default console'
+
+  if (vm.get('sessions') && vm.get('sessions').filter(c => c.get('consoleUser')).size) {
```

Comment:
please use `.isEmpty()` instead of relying on the `size` > 0 causing the condition to be true